### ISSUE TITLE
Trying to use release-it on a local git and npm package

### ISSUE THREAD

**Issue by ath0mas** [release-it/release-it#666]
Hello, I am trying to use _release-it_ on a fully local project.
But it seems a _remote repository_ is necessary, even if I try to toggle off some 'related' options (`npm.publish`, `git.requireUpstream`, `git.push`)

basic setup:
```
mkdir releaseit-foo && cd releaseit-foo
npm init private && git init && git add . && git commit -m "initial commit"
```

then trying
**$ npx release-it --dry-run**
```
$ git diff-index --quiet HEAD --
$ git rev-parse --abbrev-ref HEAD
$ git config --get branch.master.remote
$ git remote get-url origin
$ git config --get remote.origin.url
ERROR Could not get remote Git url.
Please add a remote repository.
```
**$ npx release-it --no-npm.publish --no-git.requireUpstream --no-git.push --dry-run**
```
$ git diff-index --quiet HEAD --
$ git rev-parse --abbrev-ref HEAD
$ git config --get branch.master.remote
$ git remote get-url origin
$ git config --get remote.origin.url
ERROR Could not get remote Git url.
Please add a remote repository.
```

**Comment by webpro** [comment]
I agree, this is a bug, I'll look into this shortly.

**Comment by webpro** [comment]
Thanks for the clear description and repo steps.

I've just released a fix in [v13.6.2](https://github.com/release-it/release-it/releases/tag/13.6.2).

You will need to use ` --no-git.push` and `--no-git.requireUpstream` without a remote. I will fix up the error messages later.

**Comment by ath0mas** [comment]
🚀 Let's release, and 🏁 Done

Thanks! @webpro
